generator/utils/importsResolver.ts

```diff
@@ -17,17 +17,17 @@ const GovernanceImports = [
 function generateAddressBookImports(code: string) {
   const imports: string[] = [];
   let root = '';
-  const addressBookMatch = code.match(/(AaveV[2..3][A-Z][a-z]+)\./);
+  const addressBookMatch = code.match(/(AaveV[2..3][A-Za-z]+)\./);
```

Comment:
the regex expects at least one lowercase, but in case on BNB there is non. This fixes it.